From a91634d476356a70003203753fbac4557d8b1bd7 Mon Sep 17 00:00:00 2001
From: huangyifeng <huangyifeng@eswincomputing.com>
Date: Fri, 10 Jan 2025 16:45:33 +0800
Subject: [PATCH 383/481] WIN2030-16919:fix:select HAVE_SETUP_PER_CPU_AREA if
 NUMA enable

Changelogs:
        select HAVE_SETUP_PER_CPU_AREA only if NUMA enable

Change-Id: I6608b5c59c06a473f3b11e1959d7ccbb277b1dbf
Signed-off-by: huangyifeng <huangyifeng@eswincomputing.com>
---
 arch/riscv/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/Kconfig b/arch/riscv/Kconfig
index d69a7dc8a022..02d015d8850e 100644
--- a/arch/riscv/Kconfig
+++ b/arch/riscv/Kconfig
@@ -153,7 +153,7 @@ config RISCV
 	select MODULE_SECTIONS if MODULES
 	select OF
 	select OF_EARLY_FLATTREE
-	select HAVE_SETUP_PER_CPU_AREA
+	select HAVE_SETUP_PER_CPU_AREA if NUMA
 	select OF_IRQ
 	select PCI_DOMAINS_GENERIC if PCI
 	select PCI_MSI if PCI
-- 
2.49.0

